### DISCUSSION TITLE
Please add "Probabilistic chess" variant

### DISCUSSION THREAD

**Discussion by sensorlife** [gbtami/pychess-variants#1058] [Ideas]
2 years ago I asked on lichess help to implement "Probabilistic chess" variant ( https://lichess.org/forum/lichess-feedback/feature-request-please-add-probabilistic-chess-game ).
Can you please help me to implement this variant on pychess website ?
This is amazing game!

**Comment by gbtami** [comment]
Pychess is based on https://github.com/ianfab/Fairy-Stockfish meaning only variants supported by Fairy can be added.

**Reply by sensorlife** [reply]
But I don't need to play with computer. I need to play online with humans.

**Reply by gbtami** [reply]
The thing is pychess code itself doesn't contain the variant rules (like valid moves, FEN, SAN, PGN, isCheck(), game termination rules etc-etc) at all. It all comes from Fairy-Stockfish knowledge via its Pyhon and Javascript bindings. So adding any variants is impossible if Fairy doesn't provides all of these for us.
